reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -130,6 +128,14 @@ def iterative_refinement(self, wiener_small_number=0.01, count_norm_const=1):
             .reshape(map_shape)
         )
 
+        wiener_small_numbers = IterativeRefinement.compute_ssnr(
+            self.particles, ctfs, 0.01
+        )
+        wiener_small_numbers = np.where(
```

Comment:
instead of `wiener_small_numbers == 0`, perhaps `np.isclose(wiener_small_numbers,0)` for numerical stability